tests/test_imports.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+"""Test that imports work correctly on Python 3.10+"""
+import sys
+import unittest
+
+
+class TestImports(unittest.TestCase):
+    """Test module imports work on Python 3.10+"""
+
+    def test_socks_import_no_collections_error(self):
+        """Test that PySocks can be imported without collections.Callable error"""
+        try:
+            import socks
+            self.assertTrue(hasattr(socks, 'socksocket'))
+        except ImportError as e:
+            if 'Callable' in str(e) and 'collections' in str(e):
+                self.fail(f"PySocks has collections.Callable issue (needs PySocks>=1.7.1): {e}")
+            raise
+
+    def test_collections_abc_callable_available(self):
+        """Test that collections.abc.Callable is available (Python 3.3+)"""
```

Comment:
In that case, we should also change setup.cfg `python_requires = >=3.0` into `>=3.3`